java/dbschema/src/org/netbeans/modules/dbschema/SchemaElementUtil.java

```diff
@@ -59,7 +59,7 @@ public static SchemaElement forName(String name, Object obj) {
                 String tempURL = ""; //NOI18N
                 if (schemaFO != null)
                     try {
-                        tempURL = schemaFO.getURL().toString();
+                        tempURL = schemaFO.toURL().toString();
                     } catch (Exception exc) {
```

Comment:
Is this catch still necessary? At lease I would reduce it to `RuntimeException` or is thee another checked exception that could be thrown?

Reply:
Fixed..